pkg/kubelet/cm/deviceplugin/device_plugin_stub.go

```diff
@@ -109,7 +106,8 @@ func (m *Stub) Stop() error {
 
 // Register registers the device plugin for the given resourceName with Kubelet.
 func (m *Stub) Register(kubeletEndpoint, resourceName string) error {
-	conn, err := grpc.Dial(kubeletEndpoint, grpc.WithInsecure(),
+	conn, err := grpc.Dial(kubeletEndpoint, grpc.WithInsecure(), grpc.WithBlock(),
```

Comment:
can we use the dial function here ?

Reply:
I will leave this up-to @vikaschoudhary16 to decide. Note that this part will change once we have a generic kubelet plugin model, which will use probe instead of registration.